stingray/deadtime/model.py

```diff
@@ -0,0 +1,151 @@
+from stingray.utils import njit, prange
+from math import gamma
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import log
+
+
+def r_in(td, r_0):
+    """Calculate incident countrate given dead time and detected countrate."""
+    tau = 1 / r_0
+    return 1. / (tau - td)
+
+
+def r_det(td, r_i):
+    """Calculate detected countrate given dead time and incident countrate."""
+    tau = 1 / r_i
+    return 1. / (tau + td)
+
+
+@njit()
+def factorial(n):
+    return gamma(n + 1)
+
+
+@njit()
+def Gn(x, n):
+    s = 0
+    for l in range(0, n):
+        s += (n - l) / factorial(l) * x**l
+    return np.exp(-x) * s
+
+
+@njit()
+def heaviside(x):
+    """Heaviside function.
+
+    Examples
+    --------
+    >>> heaviside(2)
+    1
+    >>> heaviside(-1)
+    0
+    """
+    if x >= 0:
+        return 1
+    else:
+        return 0
+
+
+@njit()
+def h(k, n, td, tb, tau):
```

Comment:
What does this function do? The function name isn't very descriptive, so adding a docstring might help? Also, where is it used?